src/bin/vip-config-software-get.js

```diff
@@ -29,11 +42,11 @@ command( {
 	envContext: true,
 	wildcardCommand: true,
 	format: true,
-	usage: 'vip @mysite.develop config software get <wordpress|php|nodejs|muplugins>',
+	usage,
 } )
 	.option(
 		'include',
-		`Extra information to be included. Valid values: ${ VALID_INCLUDES.join( ',' ) }`
+		`Additional data to be included. Supported values: ${ VALID_INCLUDES.join( ',' ) }`
```

Comment:
Suggested:
`Retrieve additional data of a specific type. Supported values: `
Future-proofing by suggesting that eventually users will be able to pass more than 1 value to this option.